File: Algorithmique appliquée/chapitre3/exercice6.cpp
/* programme nombres (exercice 6)
*  but : différents traitements sur des nombres
*  date : 31/10/2017
*/
#include <iostream>
#include <maBiblio.h>

using namespace std;

int main(int argc, char *argv[]){
    cout << "filou";
    char choix;
    unsigned int nb;

    choix = 'Z';
    while( choix != 'Q' && choix != 'q'){
        cout << endl << "50 premiers multiples d'un nombre....... 1 \n";
        cout << endl << "facteurs premiers d'un nombre........... 2 \n";
        cout << endl << "50 premiers nombres premiers............ 3 \n";
        cout << endl << "quitter....... Q \n";
        cin >> choix;

        if(choix == '1'){
            cout << "entrer un nombre = ";
            cin >> nb;
            multiples(nb,50);
        }
        if(choix == '2'){
            cout << "entrer un nombre = ";
            cin >> nb;
            facteursPremiers(nb);
        }
        if(choix == '3'){
            nombresPremiers(50);
        }

    }
    return 0;
}

File: Algorithmique appliquée/chapitre3/python/exercice6.py
# -*-coding:utf-8 -*
# Programme exercice 6
# différents traitements sur des nombres
# date : 31/10/2017


import os
import math
import sys
sys.path.append("..\\bibliotheque")
from maBiblio import *


# boucle sur le menu
choix = 'Z'
while choix != 'Q' and choix != 'q' :
    # afficher le menu
    print("50 premiers multiples d'un nombre ........ 1")
    print("facteurs premiers d'un nombre ............ 2")
    print("50 premiers nombres premiers ............. 3")
    print("quitter .................................. Q")
    choix = input("votre choix .............................. ")

    # 50 premiers multiples d'un nombre
    if choix == '1' :
        nb = int(input("Entrez un nombre = "))
        multiples(nb, 50)
    else :
        # facteurs premiers d'un nombre
        if choix == '2' :
            nb = int(input("Entrez un nombre = "))
            facteursPremiers(nb)
        else :
            # 50 premiers nombres premiers
            if choix == '3' :
                nombresPremiers(50)


os.system("pause")
